test/e2e/serial/tests/operator_profile_update.go

```diff
@@ -0,0 +1,282 @@
+	. "github.com/onsi/gomega"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	manifests "github.com/k8stopologyawareschedwg/deployer/pkg/manifests"
+	k8swgobjupdate "github.com/k8stopologyawareschedwg/deployer/pkg/objectupdate"
+	nrtv1alpha2 "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/apis/topology/v1alpha2"
+
+	nropv1 "github.com/openshift-kni/numaresources-operator/api/numaresourcesoperator/v1"
+	"github.com/openshift-kni/numaresources-operator/internal/wait"
+	serialconfig "github.com/openshift-kni/numaresources-operator/test/e2e/serial/config"
+	e2eclient "github.com/openshift-kni/numaresources-operator/test/utils/clients"
+	e2efixture "github.com/openshift-kni/numaresources-operator/test/utils/fixture"
+	"github.com/openshift-kni/numaresources-operator/test/utils/objects"
+)
+
+const (
+	eventReasonFailedScheduling = "FailedScheduling"
+	eventMsgInvalidTopology     = "invalid node topology data"
+)
+
+var _ = Describe("[serial][nrop_update] numaresources operator profile updates", Serial, func() {
+	var fxt *e2efixture.Fixture
+	var nrtList nrtv1alpha2.NodeResourceTopologyList
+
+	BeforeEach(func() {
+		Expect(serialconfig.Config).ToNot(BeNil())
+		Expect(serialconfig.Config.Ready()).To(BeTrue(), "NUMA fixture initialization failed")
+
+		var err error
+		fxt, err = e2efixture.Setup("e2e-test-profile-updates", serialconfig.Config.NRTList)
+		Expect(err).ToNot(HaveOccurred(), "unable to setup test fixture")
+
+		err = fxt.Client.List(context.TODO(), &nrtList)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		err := e2efixture.Teardown(fxt)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	Context("using the NUMA-aware scheduler without updated NRT data", func() {
+		// TODO The set of tests under this context are expecting that all worker nodes of the cluster
+		// be NROP-configured. this case is covered. However, if the cluster has other worker nodes (>2)
+		// and those nodes are not NROP-configured then the expected behavior of these tests will change
+		// and currently is missing and likely to be addressed in https://issues.redhat.com/browse/CNF-10552.
+		var testPod *corev1.Pod
+		nropObjInitial := &nropv1.NUMAResourcesOperator{}
+		nroKey := objects.NROObjectKey()
+
+		BeforeEach(func() {
+			By("disable RTE functionality hence publishing updated NRT data")
+			err := fxt.Client.Get(context.TODO(), nroKey, nropObjInitial)
+			Expect(err).ToNot(HaveOccurred(), "cannot get %q from the cluster", nroKey.String())
+
+			updatedNROPObj := nropObjInitial.DeepCopy()
+			infoRefreshPauseMode := nropv1.InfoRefreshPauseEnabled
+			//TODO adapt the existence of multiple node groups,https://issues.redhat.com/browse/CNF-10552
+			updatedNROPObj.Spec.NodeGroups[0].Config = &nropv1.NodeGroupConfig{
+				InfoRefreshPause: &infoRefreshPauseMode,
+			}
+
+			err = fxt.Client.Update(context.TODO(), updatedNROPObj)
```

Comment:
updates should always be done in a loop (like `Eventually`). Updating the NROP objects is unlikely to fail, but still